config/locales/custom/es/admin.yml

```diff
@@ -107,6 +107,10 @@ es:
           film_library_limit: "Límite de películas"
           film_library_limit_down: "Si no se introduce limite o se indica que es '0', no hay limite de propuestas"
           film_library_admin: "Sólo administradores"
+      questions:
+        table:
+          downvotes: "Votos negativos"
+          upvotes: "Votos positivos"
```

Comment:
Since people are not really voting in favor or against the question, how about "Votos pregunta clara" and "Votos pregunta confusa"? It's not that I love this idea either, so feel free to come up with something better :wink:.